### PR TITLE
Add quick start pages for STJ tools

### PR DESCRIPTION
**Pull request by clsackett**
Add quick start pages for STJ tools:
- Gene Expression Clustering
- OncoMatrix

### CODE REVIEW

**Comment by wteouchicago** [comment]
looks ok to me!

do we want to update other areas in this PR to account for the fact that the gene expression page is in, e.g.?
- https://github.com/NCI-GDC/gdc-docs/blob/develop-v2/docs/Data_Portal/Users_Guide/analysis_center.md?plain=1#L23
- https://github.com/NCI-GDC/gdc-docs/blob/develop-v2/Data_Portal_UG.yml
- https://github.com/NCI-GDC/gdc-docs/blob/develop-v2/mkdocs.yml

**Comment by wteouchicago** [comment]
LGTM! 👍

File: docs/Data_Portal/Users_Guide/oncomatrix_quick_start.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+[![Analysis Tools with OncoMatrix Card](./images/oncomatrix/1-analysis_center.png)](./images/oncomatrix/1-analysis_center.png 'Click to see the full image.')
+
+Users can view publicly available genes as well as login with credentials to access controlled data.
+
+## OncoMatrix Features
+
+There are three main panels in the OncoMatrix tool: [control panel](#control-panel), [matrix plot](#matrix-plot), and [legend panel](#legend-panel).
+
+[![OncoMatrix Overview](/images/oncomatrix_overview.png)](/images/oncomatrix_overview.png 'Click to see the full image.')
```
[the remark this answers is not in the document]
Comment (clsackett) [reply docs/Data_Portal/Users_Guide/oncomatrix_quick_start.md]: @wteouchicago  I just removed the slashes - can you see if the images appear for you now?

[the remark this answers is not in the document]
Comment (wteouchicago) [reply docs/Data_Portal/Users_Guide/oncomatrix_quick_start.md]: things are showing up now! 👍

**Comment by wteouchicago** [comment]
qqn: where is the oncomatrix quick start going (vs the "regular" oncomatrix guide)? possibly i miss this somewhere, but do we need to add a link to it or something?

**Comment by clsackett** [comment]
Sorry @wteouchicago - forgot that you're lacking context! Bill and I discussed making quick start guides for some of the STJ tools so that they align better with other GDC docs. TBD on what we're planning to do with the full STJ documentation, but for right now I'm going to make quick start pages for these tools: oncomatrix, PP lollipop, sequence reads

**Comment by wteouchicago** [comment]
> Sorry @wteouchicago - forgot that you're lacking context! Bill and I discussed making quick start guides for some of the STJ tools so that they align better with other GDC docs. TBD on what we're planning to do with the full STJ documentation, but for right now I'm going to make quick start pages for these tools: oncomatrix, PP lollipop, sequence reads

ahhh, ok!

**Review by wteouchicago — APPROVED** [review]

**Pull request merged**